### ISSUE TITLE
With one favourite, it should be the default view

### ISSUE THREAD

**Issue by pihvi** [suominentoni/nearest-departures#3]
The nearest stop to my home is the only and daily use for me. So in practise when I open the app, I first click favourites and then I click the only item on that list to get what I want. In my case it would be nice to see that when I open the app. What do you think if one had only one favourite, it would be opened by default on app open?

**Comment by suominentoni** [comment]
Sounds good!

Do you think this behaviour should be communicated to the user somehow? And I mean in the app itself, not just the release notes. I'm afraid adding this feature without any guidance could lead to confusion.

**Comment by pihvi** [comment]
First time the app opens the only favourite by default, it would be nice to get a one time info that this happens when you have just one favourite.

Next step would be to make it configurable i.e. user can choose what is the default opening view.

**Comment by pihvi** [comment]
But in order to have the favourite, user must already know how the app works and is navigated. So when the one favourite is opened by default, I think it's clear where you are and how you can navigate to the main view or favourites list, so I don't think the change communication to user is a must..

**Comment by suominentoni** [comment]
Yes, I'm sure the user can easily figure out where he is. The problem is that he most probably will not know _why_ he is there. Hence the guidance. 

I think I'd also like to make it configurable from the start.

Interested in coming up with a PR? :)

**Comment by pihvi** [comment]
well.. how should I put it.. yeeeeees, but.. it might take a while. so don't hold your breath. now we has the specs about at least!

**Comment by suominentoni** [comment]
No worries! If you end up starting working on it, let me know. I'll do the same.
